src/mutate-errors-to-todos.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  TodoData,
+  todoFilePathFor,
+  _buildTodoDatum,
```

Comment:
If we need to use this, I can rename it so it's not implicitly private.

Reply:
I think it's useful to explicitly export this one since it's useful to get the file hash given a result and a message. If you decide to rename it let me know.